.travis.yml

```diff
@@ -3,4 +3,3 @@ rvm:
 - 2.1
 - 1.9.3
 - jruby
-- jruby-head
```

Comment:
It looks like it was using jruby-9.0.1.0-SNAPSHOT which seems like is a little more edge than we need?